### PR TITLE
(#456) fix update of hostname of Codenvy on-prem with machine nodes

### PR DESCRIPTION
**Pull request by dmytro-ndp**
List of points:
1) fix absolute value of **certname** property in _/etc/puppet/puppet.conf_ on machine node;
2) update _/etc/puppet/autosign.conf_ with new machine node hostnames on master;
3) update *$swarm_node* variable in _/etc/puppet/manifests/nodes/codenvy/codenvy.pp_ on master  with new machine node hostnames;
4) check if existed nodes with new hostname are available before changing hostname.

@tolusha: please, review this request.

Signed-off-by: Dmytro Nochevnov <dnochevnov@codenvy.com>

### CODE REVIEW

File: cdec/installation-manager-core/src/main/java/com/codenvy/im/managers/helper/NodeManagerHelper.java [hunk 1]
```diff
@@ -91,6 +95,32 @@ public Command getValidatePuppetMasterAccessibilityCommand(String puppetMasterNo
         return CommandLibrary.createCheckRemotePortOpenedCommand(puppetMasterNodeDns, 8140, node);
     }
```
Comment (skabashnyuk) [inline cdec/installation-manager-core/src/main/java/com/codenvy/im/managers/helper/NodeManagerHelper.java]: javadoc

File: cdec/installation-manager-core/src/main/java/com/codenvy/im/commands/CommandLibrary.java [hunk 2]
```diff
@@ -533,7 +533,7 @@ public static Command createCheckRemotePortOpenedCommand(String remoteHost, int
      * @see "http://www.unix.com/302910488-post2.html?s=c351328813d6bc7a509c0a5dee5dc9e7"
      */
     private static String getCheckRemotePortOpenedCommand(String remoteHost, int remotePort) {
-        return format("2>/dev/null >/dev/tcp/%s/%s", remoteHost, remotePort);
+        return format("sleep 3 && echo >/dev/tcp/%s/%s", remoteHost, remotePort);
```
Comment (tolusha) [inline cdec/installation-manager-core/src/main/java/com/codenvy/im/commands/CommandLibrary.java]: Why do we need sleep command ?

**Comment by tolusha** [comment]
+1

File: cdec/installation-manager-core/src/main/java/com/codenvy/im/commands/CommandLibrary.java (hunk 2 above)
Comment (dmytro-ndp) [inline cdec/installation-manager-core/src/main/java/com/codenvy/im/commands/CommandLibrary.java]: because of without sleep it fails

**Pull request merged**